src/css/profile/mobile/common/listview.less

```diff
@@ -25,13 +25,34 @@ tau-expandable {
 	 * because this widget has canvas which has to be visible outside the listview
 	 */
 	// overflow: hidden;
-	margin: 0;
+	margin: auto;
+
+	// List width for Tablet
+	@media (orientation: portrait) and (min-width: 673px) {
+		width: 90%;
+	}
+	@media (orientation: landscape) and (min-width: 960px) {
+		width: 75%;
+	}
+
```

Comment:
I have a question.
Just if orientation is portrait and min-width is 700px, how about is the width ??

just for example,
@media (min-width: 600px) and (max-width: 959px) and (min-height: 0px) and (max-height: 479px)
=> Mobile Landscape

Reply:
Just if orientation is portrait and min-width is 700px, how about is the width ??
>> It will be working as the tablet.
@media (min-width: 600px) and (max-width: 959px) and (min-height: 0px) and (max-height: 479px)
=> Mobile Landscape
=> I set the default value as the mobile.
Default: marin auto;
Whan I need to update the value with media query then, I added the @media